docusaurus/docs/contributing/index.mdx

```diff
@@ -2,13 +2,155 @@ import DocCardList from '@theme/DocCardList';
 
 # Contributing
 
-**Step-by-step technical instructions for contributors**
-
 This section explains how to contribute to the ATT&CK Data Model repository.
-It complements the high-level `CONTRIBUTING.md` in the root of the repository by focusing on specific developer tasks.
+This includes the general contribution guidelines in addition to step-by-step technical instructions for specific developer tasks.
 
 ## Topics
 
 <DocCardList />
 
+## How to Contribute
+
+We don't have a formal process or Standard Operating Procedure (SOP) for accepting changes. However, there are several ways you can contribute:
+
+1. **Open Issues**: If you encounter bugs or have feature requests, please open an issue using the templates provided below.
+2. **Submit Pull Requests**: Feel free to fork the repository, make changes, and submit a pull request. We will review your changes as necessary.
+
+## Pull Request Guidelines
+
+When submitting a pull request:
+
+1. Fork the repository and create your branch from `main`.
+2. Ensure your code adheres to the existing style of the project to maintain consistency.
+3. Include comments in your code where necessary.
+4. Update the README.md with details of changes to the interface, if applicable.
+5. Increase the version numbers in any examples files and the README.md to the new version that this Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+6. Include a description of your changes and why they're necessary.
+
+## Developer Workflow
+
+To maintain code quality and consistency, we use **ESLint**, **Prettier**, and **Husky** as part of the development workflow. These tools are installed and configured by the [Developer Setup](#developer-setup) steps above. Below is an overview of how each tool is configured and how it affects the contribution process:
```

Comment:
```suggestion
To maintain code quality and consistency, we use **ESLint**, **Prettier**, and **Husky** as part of the development workflow. These tools are installed and configured by the [Developer Setup](https://mitre-attack.github.io/attack-data-model/docs/contributing/dev-setup) steps. Below is an overview of how each tool is configured and how it affects the contribution process:
```

This is the right link now, but it talks about **ESLint**, **Prettier**, and **Husky** here - but not on the page linked.